### DISCUSSION TITLE
SRA file to FASTQ

### DISCUSSION THREAD

**Discussion by dippelmax** [sr320/course-fish546-2021#72] [Q&A]
I am attempting to get my data from SRA format to FASTQ. It is from the NCBI website and here is the link: https://www.ncbi.nlm.nih.gov/sra/SRX3265265[accn] We have been working on it in class and have run into some challenges.

**Comment by kubu4** [comment]
Here's my guide that I wrote last year on how to use the SRA Toolkit:

https://robertslab.github.io/sams-notebook/2020/05/21/Tutorial-SRA-Toolkit-for-Data-Retrieval-and-Conversion-to-FastQ.html

Be aware, it neglects to cover the configuration, which is a bit confusing.

Here are the configuration basics:

1. Use the Space Bar or Enter key to make your selection. Your selection can be highlighted using the mouse cursor. Your selection is designated by a red vertical bar.
2. Use the red letters to pick a selection. If you wanted to use the "exit" menu option, the letter "x" will be red. Type "x" to exit.
3. Under the "CACHE" tab, choose a "local repository" where you want your files saved.
4. Use the "save" option.
5. Use the "exit" option.

Should be good to go.

Run the test command (you might need to specify the full to the `fastq-dump` program):

`fastq-dump --stdout -X 2 SRR390728`

**Comment by sr320** [comment]
Note this worked also .... https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump
